extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -19,8 +19,19 @@ amp-sticky-ad {
   text-align:center;
   bottom: 0 !important;
   left: 0;
-  z-index: 1;
   width: 100%;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  box-sizing: border-box;
+}
+
+.-amp-sticky-ad-layout {
+  display: flex;
+  flex-direction: row;
```

Comment:
I was indeed trying to do left to right here but I agree it doesn't make sense to put ads in same line. 
I changed it to column.